kubernetes/config/kube_config.c

```diff
@@ -167,7 +186,7 @@ static int kubeconfig_exec(kubeconfig_property_t * current_user)
         goto end;
     }
 
-  end:
+    end:
```

Comment:
Could you please keep the original indentation here ?

Reply:
Fixed